sklearn/decomposition/tests/test_dict_learning.py

```diff
@@ -57,6 +57,54 @@ def test_dict_learning_overcomplete():
     assert dico.components_.shape == (n_components, n_features)
 
 
+def test_max_iter():
+    def ricker_function(resolution, center, width):
+        """Discrete sub-sampled Ricker (Mexican hat) wavelet"""
+        x = np.linspace(0, resolution - 1, resolution)
+        x = ((2 / ((np.sqrt(3 * width) * np.pi ** 1 / 4)))
```

Comment:
I think you must need parentheses around the 1/4... ** takes precedence. Perhaps use 0.25 instead of 1/4. You can remove some parentheses by changing * to /.

Reply:
I didn't write these. They're copy pasted from the failing example (`plot_sparse_coding.py`).

But I'll try to simplify both then.